swan-lake/development-tutorials/ballerina-central/publish-packages-to-ballerina-central.md

```diff
@@ -167,3 +167,21 @@ To reverse the deprecation of a package, execute the same command with the `--un
 ```
 $ bal deprecate <org-name>/<package-name>:<version> --undo
 ```
+
+### Deprecate all the versions of a published package
+
+To change the name of a published package, you can mark all the versions of the existing package as deprecated and then publish the package under a new name.
+
+The package owner can run the following command. An optional deprecation message can also be included.
+
+```
+$ bal deprecate <org-name>/<package-name> --message <deprecation-message>
+```
+
+If the deprecated package is in use, a warning message containing the provided deprecation message will be shown during the project build.
```

Comment:
```suggestion
If the deprecated package is in use, a warning message containing the provided deprecation message will be shown during the project build.
```
Should this also be an info or tip section?

Reply:
Fixed.